source/magic.cpp

```diff
@@ -4964,7 +4969,7 @@ void CMagic::LoadScript( void )
 									auto ssecs = oldstrutil::sections( data, " " );
 									if( ssecs.size() > 1 )
 									{
-										spells[i].Flags((( static_cast<UI08>( std::stoul( oldstrutil::trim( oldstrutil::removeTrailing( ssecs[0], "//" )), nullptr, 16 ))) << 8 ) ||
+										spells[i].Flags((( static_cast<UI08>( std::stoul( oldstrutil::trim( oldstrutil::removeTrailing( ssecs[0], "//" )), nullptr, 16 ))) < 8 ) ||
```

Comment:
I think you need to revert this.  You've changed a deliberate bit shift to a comparison, and I don't think you'll get the same outcome.

I also suspect it's supposed to be an | not || so that the two fields combine

Reply:
Good catch. This has been corrected.